src/player.rs

```diff
@@ -399,10 +398,9 @@ impl Default for GameInfo {
             time: 0,
             ping: 0,
             loss: 0,
-            state: PlayerState::Spawning,
+            state: PlayerState::Active,
```

Comment:
Why is this defaulted to `Active` now?

Reply:
I suppose it would be a better default for 100-Player servers where half of the players won't ever get set to `Active` due to being cut out of the status output. If we want to default it to `Active` though it should also be changed on line 480 when acknowledging the player (which is what the original PR did in the first place, but I preferred `Spawning` at the time).

Reply:
Yeah, I defaulted to active to keep the interface clean on a 100 player server.

On searching the repo there's also regexes.rs L142; I'll update that as well.

Reply:
Shouldn't matter for the regex bc if it actually parses one out for that player then it'll be valid.

Reply:
Did it for consistency anyway. Don't want anyone getting the wrong idea later down the road.

Reply:
Ah well if it's already done